### DISCUSSION TITLE
music.youtube.com

### DISCUSSION THREAD

**Discussion by gbroadbent67** [deepjyoti30/ytmdl#180] [Q&A]
By default the program searches youtube.com, how can it be updated to search music.youtube.com ?

**Comment by deepjyoti30** [comment]
@gbroadbent67 That would be a bit tough. Currently the app uses a library called `youtube-search` to search YouTube. For searching YouTube there are a lot more libraries. However, YouTube Music doesn't have as many libraries and adding some scraping logic to search would make the program very slow.

If there would be a library to do that, I would love to add support for both YouTube and YouTube Music.
